firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,59 @@
+Scenario:The Util get_byte function correctly gets bytes from the given position
+   GIVEN("The util get_byte function")
+       WHEN("the get_byte function is called on 0x86 ")
+          THEN(""The value at the least significant byte position is "0x0a'")
+   GIVEN("The util get_byte function " )
+       WHEN("the get_byte function is called on 0x860a")  
+       THEN(""The value at the least significant byte position is "0x0a")
+       THEN("The value at the most signficant postiion is 0x86")
+
+
+   GIVEN("The util get_byte function'")
+       WHEN("the get_byte function is called on 0x86010f0a")
+       THEN("value is as expected ")
```

Comment:
This `GIVEN` should have four `WHEN`-`THEN` pairs: the first `WHEN`+`THEN` is for byte index 0, the second `WHEN`+`THEN` is for byte index 1, etc.